src/twisted/newsfragments/10345.misc

```diff
@@ -0,0 +1 @@
+Fix SendmailTests for python 3.11.
```

Comment:
For .misc, you can leave the file empty as it will not be visible in the release note.

The content of this file should not be a commit message :)

We already have git for that.

Here we put all the "commit" information for people using Twisted.